airflow/data/headers.yml

```diff
@@ -78,6 +78,12 @@
         - gtfs_rt_vehicle_positions_url
         - gtfs_rt_service_alerts_url
         - gtfs_rt_trip_updates_url
+    - itp_id: 280 # San Francisco Bay Ferry
+      url_number: 0
+      rt_urls:
+        - gtfs_rt_vehicle_positions_url
+        - gtfs_rt_service_alerts_url
```

Comment:
per discussion on Slack, let's remove this since we don't need to use the Swiftly key here